docs/introduction/index.rst

```diff
@@ -115,6 +120,8 @@ accidental data duplication as much as possible.
 History
 -------
 
+- 2018.08.01. **Renku** Component Architecture is solidifying now to a more long-term design
```

Comment:
I wand to add a line here when we actually do a release, so I think we can remove for now